src/metabase/api/common.clj

```diff
@@ -216,23 +218,23 @@
 
 ;; #### GENERIC 403 RESPONSE HELPERS
 ;; If you can't be bothered to write a custom error message
-(def ^:private generic-403
-  [403 (tru "You don''t have permissions to do that.")])
+(defn- generic-403 []
+  [403 (str (tru "You don''t have permissions to do that."))])
```

Comment:
wouldn't the Cheshire encoders encode this automatically?

Reply:
I believe for this kind of exception we return a `text/plain` string error message, not a JSON response, so Cheshire encoders wouldn't kick in.

I'll double check that though.

Reply:
Here's where we take that String and convert it to an exception https://github.com/metabase/metabase/pull/8452/files#diff-5ccc18379bf70c31ca9e4caf301a5544R81. There's no reason why we couldn't call `str` at that central place though. I'll switch to calling it there.

Reply:
Even better, that `check` call is already using `ui18n/ex-info`, so it already does the right thing. I can remove the `str` call entirely and it will work as-is.